src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 server.port=8080
-spring.datasource.url=jdbc:postgresql://db:5432/mongen_db
+spring.datasource.url=jdbc:postgresql://localhost:5432/mongen_db
```

Comment:
would this be related to the docker issue?

Reply:
ops, this change shouldn't be here, the realtime compilation is not working, each time is needed to do a `mvn clean install -DskipTests` to reload, in one moment I lost db connection, so I changed the URL like is shown here. I will continue looking at what the issue with hot reload is